### PR TITLE
Data base

### PR DESCRIPTION
**Pull request by girma336**
- In this projects, i use a relational database to create the data structure for a vet clinic. You will start with one table, and step by step.
- [ ]  Added updated data.sql , schema.sql and query.sql file.

Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
both_tru_wei

Find all animals that are neutered.
nue_tru

Find all animals not named Gabumon.
name_not_gu

List name and escape attempts of animals that weigh more than 10.5kg
weight_10

List the date of birth of all animals named either "Agumon" or "Pikachu".
name_in

List the name of all animals born between 2016 and 2019.
born_date

Find all animals whose name ends in "mon".
animal_end_mon

All list of animal table
all_animals

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,12 +214,6 @@ Example:
...
 
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
```
Comment (Munsa1) [inline README.md]: - [ ] The readme file is missing a `future feature's` section. This section should add some of the objectives of future milestones, kindly fix this

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name like '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```
Comment (Munsa1) [inline queries.sql]: - [ ] The date shouild be between `'2016-01-01'` and `'2019/31/12'`, kindly fix the ending date.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @girma336, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Abenezer-Tilahun) [reply README.md]: Fixed.

File: queries.sql (hunk 2 above)
Comment (Abenezer-Tilahun) [reply queries.sql]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @girma336 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**